### PR TITLE
Fixes tests

### PR DESCRIPTION
**Pull request by bmw**
Fixes failing test case due to dependency changes.

### CODE REVIEW

**Comment by bmw** [comment]
Integration tests failed :(

**Comment by schoen** [comment]
Looks good, thanks for fixing the tests!

File: tests/boulder-start.sh [hunk 1]
```diff
@@ -5,5 +5,6 @@ export GOPATH="${GOPATH:-/tmp/go}"
 
 go get -d github.com/letsencrypt/boulder
 cd $GOPATH/src/github.com/letsencrypt/boulder
+./test/create_db.sh
```
Comment (kuba) [inline tests/boulder-start.sh]: `create_db.sh` is not included in the repo, @bmw

Comment (bmw) [inline tests/boulder-start.sh]: It's in Boulder's repo: https://github.com/letsencrypt/boulder/blob/master/test/create_db.sh

**Pull request merged**